src/rulesets/oas2/index.json

```diff
@@ -90,6 +90,36 @@
         "operation"
       ]
     },
+    "valid-example-in-parameters": {
+      "description": "Examples must be valid against their defined schema.",
+      "message": "{{error}}",
+      "recommended": true,
```

Comment:
Does the rule have to be recommended? @philsturgeon 
It's expensive, so if we can have it disabled by default, that'd be great.
If you find it particularly useful, I'm fine with leaving it as is.

Reply:
This is a pretty important feature, people have reported spectral as broken when its not run, so leaving it to only people who need to know about it and turn it on seems... unideal. People can disable slow rules if they want but honestly Studio is probably one of the few scenarios where speed matters?  (That comment may not age well, but for right now...)

Reply:
> People can disable slow rules if they want but honestly Studio is probably one of the few scenarios where speed matters? 

Yeah, it's because of the UI. In case of CI it usually doesn't really matter whether linting takes 5 seconds or 20 seconds... unless you run out of memory which is what the issue is all about.
So in the end, if one lints a huge document, such as Sendgrid or Stripe, it **will** matter.

Reply:
Ok, lets get this feature in and it can be a bit slow, as we know we are working on #437 anyway.

Reply:
Either way, let's just merge it.
I'll create a follow up ticket for this feature today and will try to address it ASAP

Reply:
If anyone has their computer catch on fire im not paying the bill, but hopefully they can give us some beefy description docs to play with and we can use them for benchmarking.

Reply:
> If anyone has their computer catch on fire im not paying the bill, but hopefully they can give us some beefy description docs to play with and we can use them for benchmarking.

:laughing: we should be fine.
4.2 or 5.0 will certainly come with optimized performance, and in the meantime one can always disable the rules.